ci/test.sh

```diff
@@ -31,10 +31,10 @@ cd rdr_service
 
 # Shouldn't need this anymore
 # ./tools/install_config.sh --config=config/config_dev.json --update
-./tools/setup_local_database.sh --db_user root --db_name rdr
+# ./tools/setup_local_database.sh --db_user root --db_name rdr
```

Comment:
dont we still need this ?

Reply:
If the unit tests all use the 9306 db instance, we don't.  I believe commenting it out will let us catch tests that try to use the 3306 db instance.  Using the 3306 db instance would indicate that something is wrong that needs correcting.

Reply:
ok, isnt this how CI runs its tests though ?

Reply:
It runs them the same way as we do, it spins up a db instance on port 9306.